### DISCUSSION TITLE
T_CS pin

### DISCUSSION THREAD

**Discussion by gnlinuz** [Bodmer/TFT_eSPI#2252] [Q&A - Touch]
Hi Bodmer,

Thank you for your lovely library, it was the only one that made my esp32 and tft work. Most of the examples are working fine, then I had some issues with "#warning >>>>------>> TOUCH_CS pin not defined, TFT_eSPI touch functions will not be available!"
I read some of your replies and in order to avoid the above error you need to edit the lib and enable the T_CS pin. I have this tft3.5 inch "bought from amazon" that has from one side pins to power the lcd and the pins for LCD_RD LCD_WR etc and from the other side the D0-D7 pins with SD_SCK SD_DI etc for the SD-CARD but there is no such  pin T_CS.(the tft is like a shield for arduino very popular) I know that touch is working under arduino uno and some different lib but how can I enable the touch mode with your lib since there is no pin T_CS. Is there any other option to enable the touch with those tft? Thank you for your time and all your efforts!!!!! (I don't know if it is allowed to upload a photo of the shield)

**Comment by gnlinuz** [comment]
![371px-MAR3502-20](https://user-images.githubusercontent.com/32981388/207314231-07f7f8b8-48d2-420b-a67b-0dd544358deb.jpg)

**Comment by Bodmer** [comment]
Warnings can be disabled by including:
```
#define DISABLE_ALL_LIBRARY_WARNINGS
```
in the setup file.

The screen has an resistance analogue interface which is not supported by TFT_eSPI. See [Readme](https://github.com/Bodmer/TFT_eSPI#8-bit-parallel-support)  for details of use with a third party library.

**Reply by martinklr** [reply]
Hi Bodmer, is it a good idea simply to #define TOUCH_CS -1, when a touch screen is not needed / implemented? I don't want to disable all warnings of the library. Thank you.
